specutils/tests/test_spectrum1d.py

```diff
@@ -318,15 +318,15 @@ def test_wcs_transformations():
     spec = Spectrum1D(spectral_axis=np.arange(1, 50) * u.nm,
                       flux=np.ones(49) * u.Jy)
 
-    pix_axis = spec.wcs.world_to_pixel(np.arange(20, 30) * u.nm)
+    pix_axis = spec.wcs.world_to_pixel_values(np.arange(20, 30) * u.nm)
```

Comment:
`FAILED ../../specutils/tests/test_spectrum1d.py::test_wcs_transformations - AttributeError: 'numpy.ndarray' object has no attribute 'to_value'`

I don't think this patch is correct?

Reply:
Yeah I'll have to adjust things. It's because units have been passed in.

Reply:
spacetelescope/gwcs#457 also corrects things so that the `*_values` can accept quantity inputs. GWCS was inconsistent about this. For now its better to just wrap it in a `SpectralCoord` object like it should be for the function as it is used in the test.